### ISSUE TITLE
Vanilla Expanded Framework makes the extractor angry 🙃 

### ISSUE THREAD

**Issue by modulozero** [zzzz465/rwxml-language-server#137]
Stumbled upon this while trying to use a project with [VEF](https://steamcommunity.com/sharedfiles/filedetails/?id=2023507013) in dependencies. As far as I can tell, this is [this specific version of VFECore.dll](https://github.com/AndroidQuazar/VanillaExpandedFramework/blob/0dd36429a8feb41a129be47c1582aa13e39482a0/1.3/Assemblies/VFECore.dll), based on MD5 comparison to what's on my drive.

```
extractor $ ./extractor.exe "c:\Steam\steamapps\workshop\content\294100\2023507013\1.3\Assemblies\VFECore.dll"  "C:\Steam\steamapps\common\RimWorld\RimWorldWin64_Data\Managed"
2022-06-14 18:19:50,799 Extractor [INFO]: Extracting data from
2022-06-14 18:19:50,804 Extractor [INFO]: c:\Steam\steamapps\workshop\content\294100\2023507013\1.3\Assemblies\VFECore.dll
2022-06-14 18:19:50,804 Extractor [INFO]: C:\Steam\steamapps\common\RimWorld\RimWorldWin64_Data\Managed
2022-06-14 18:19:50,835 Extractor [WARN]: System.IO.FileLoadException: Could not load file or assembly 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies. Strong name validation failed. (Exception from HRESULT: 0x8013141A)
File name: 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' ---> System.Security.SecurityException: Strong name validation failed. (Exception from HRESULT: 0x8013141A)
The Zone of the assembly that failed was:
MyComputer
   at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadFrom(String assemblyFile, Evidence securityEvidence, Byte[] hashValue, AssemblyHashAlgorithm hashAlgorithm, Boolean forIntrospection, Boolean suppressSecurityChecks, StackCrawlMark& stackMark)
   at System.Reflection.Assembly.UnsafeLoadFrom(String assemblyFile)
   at extractor.AssemblyLoader._Load(String path) in /home/runner/work/rwxml-language-server/rwxml-language-server/extractor/extractor/AssemblyLoader.cs:line 110
   at extractor.AssemblyLoader.Load(IEnumerable`1 _paths) in /home/runner/work/rwxml-language-server/rwxml-language-server/extractor/extractor/AssemblyLoader.cs:line 81


2022-06-14 18:19:50,846 Extractor [WARN]: System.IO.FileLoadException: Could not load file or assembly 'System.ServiceModel.Internals, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies. Strong name validation failed. (Exception from HRESULT: 0x8013141A)
File name: 'System.ServiceModel.Internals, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' ---> System.Security.SecurityException: Strong name validation failed. (Exception from HRESULT: 0x8013141A)
The Zone of the assembly that failed was:
MyComputer
   at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadFrom(String assemblyFile, Evidence securityEvidence, Byte[] hashValue, AssemblyHashAlgorithm hashAlgorithm, Boolean forIntrospection, Boolean suppressSecurityChecks, StackCrawlMark& stackMark)
   at System.Reflection.Assembly.UnsafeLoadFrom(String assemblyFile)
   at extractor.AssemblyLoader._Load(String path) in /home/runner/work/rwxml-language-server/rwxml-language-server/extractor/extractor/AssemblyLoader.cs:line 110
   at extractor.AssemblyLoader.Load(IEnumerable`1 _paths) in /home/runner/work/rwxml-language-server/rwxml-language-server/extractor/extractor/AssemblyLoader.cs:line 81


2022-06-14 18:19:50,885 Extractor [INFO]: extracting data...
2022-06-14 18:19:50,887 Extractor [INFO]: Extracting data from VFECore, Version=1.1.7.0, Culture=neutral, PublicKeyToken=null
2022-06-14 18:19:50,914 Extractor [ERROR]: DLL [0Harmony, Version=2.2.1.0, Culture=neutral, PublicKeyToken=null] was not provided to AssemblyReference, you should include it.

Unhandled Exception: System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at extractor.RawTypeInfo..ctor(Type T) in /home/runner/work/rwxml-language-server/rwxml-language-server/extractor/extractor/RawTypeInfo.cs:line 81
   at extractor.Extractor.CollectData_BFS(IEnumerable`1 _types) in /home/runner/work/rwxml-language-server/rwxml-language-server/extractor/extractor/Extractor.cs:line 84
   at extractor.Extractor.parse(IEnumerable`1 assemblies) in /home/runner/work/rwxml-language-server/rwxml-language-server/extractor/extractor/Extractor.cs:line 54
   at extractor.Program.<>c__DisplayClass1_0.<Main>b__0(ProgramOptions option) in /home/runner/work/rwxml-language-server/rwxml-language-server/extractor/extractor/Program.cs:line 78
   at CommandLine.ParserResultExtensions.WithParsed[T](ParserResult`1 result, Action`1 action)
   at extractor.Program.Main(String[] args) in /home/runner/work/rwxml-language-server/rwxml-language-server/extractor/extractor/Program.cs:line 58
```

**Comment by zzzz465** [comment]
hi, this will be fixed in version 0.38.0

**Comment by modulozero** [comment]
Thank you!
